app-backend/migrations/src_migrations/main/scala/48.scala

```diff
@@ -0,0 +1,34 @@
+import slick.driver.PostgresDriver.api._
+import com.liyaos.forklift.slick.SqlMigration
+
+object M48 {
+  RFMigrations.migrations = RFMigrations.migrations :+ SqlMigration(48)(List(
+    sqlu"""
+ALTER TABLE users
+  ADD COLUMN
+    organization_id
+      UUID
+      DEFAULT 'dfac6307-b5ef-43f7-beda-b9f208bb7726'
```

Comment:
I could rewrite this to take values from the existing table before 'nixing it, but this was just easier. In a brief conversation, @notthatbreezy had mentioned that currently there are _no_ users at all, so we can just associate all users with the `Public` organization.

What concerned me more initially was setting this to a hardcoded GUID, which could potentially be exploited at some point and thus would be hard to change. However, this hardcoded value has precedence in the [original migration](https://github.com/azavea/raster-foundry/blob/develop/app-backend/migrations/src_migrations/main/scala/6.scala) and thus will, for the time being, work as it should in all environments.